control/trajectory_follower/src/qp_solver/qp_solver_osqp.cpp

```diff
@@ -65,6 +65,11 @@ bool8_t QPSolverOSQP::solve(
     &U_osqp[0],
     static_cast<Eigen::Index>(U_osqp.size()), 1);
 
+  const int64_t status_val = std::get<3>(result);
+  if (status_val != 1) {
+    RCLCPP_WARN(logger_, "optimization failed : %s", osqpsolver_.getStatusMessage().c_str());
```

Comment:
Should the function `return false` here ?

Reply:
@maxime-clem I put this code here. In previous, we used the computed command even if it failed.  I just put the warning for debugging since adding "return false" may change the system behavior (and we didn'T have enough time for vehicle testing). 
Of course, we should return the false here, but there is no appropriate error handing so far, so I just have kept it as it is.

At least we should write TODO and fix it after porting is done.

Reply:
addressed here https://github.com/tier4/AutowareArchitectureProposal.iv/pull/708/commits/732686d5e1554a2673a6e443886959cf5631a49b